addons/pause/addon.json

```diff
@@ -25,5 +25,18 @@
     }
   ],
   "tags": ["editor", "projectPlayer", "recommended"],
-  "enabledByDefault": false
+  "enabledByDefault": false,
+  "settings": [
+    {
+      "name": "Alt+X keybind",
```

Comment:
Better terms?

Reply:
> Better terms?

Any ideas? How about "Alt+X to pause/resume"

Reply:
I don't think this needs to be optional.

Reply:
Well, it prevents any default actions so... maybe it should just be enabled by default?